fog/view/connection/src/bin/client.rs

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2018-2023 The MobileCoin Foundation
+
+use clap::Parser;
+use grpcio::EnvBuilder;
+use mc_attest_verifier::{Verifier, DEBUG_ENCLAVE};
+use mc_common::logger::{create_root_logger, log, Logger};
+use mc_fog_uri::{ConnectionUri, FogViewUri};
+use mc_fog_view_connection::FogViewGrpcClient;
+use mc_fog_view_protocol::FogViewConnection;
+use mc_util_grpc::GrpcRetryConfig;
+use std::sync::Arc;
+
+/// Parser for fog-view-client binary
+#[derive(Clone, Debug, Parser)]
+#[clap(
+    name = "fog-view-client",
+    about = "Test client for Fog view service.",
+    version
+)]
+struct Config {
+    /// Fog View service URI.
+    #[clap(long, env = "MC_FOG_VIEW")]
+    pub fog_view: FogViewUri,
+}
+
+fn main() {
+    // Logging must go to stderr to not interfere with STDOUT
+    std::env::set_var("MC_LOG_STDERR", "1");
```

Comment:
:question: What is the negative affect this prevents? I commented it out and saw nothing negative.

Reply:
Initially I was thinking to make the final log lines be a json blob on stdout, but I ended up not doing that.
I still think it's usually good for tools to send logging to stderr.